Source/Core/VideoCommon/PixelEngine.cpp

```diff
@@ -273,26 +251,22 @@ static void SetFinish_OnMainThread(u64 userdata, s64 cyclesLate)
 // THIS IS EXECUTED FROM VIDEO THREAD
 void SetToken(const u16 _token, const int _bSetTokenAcknowledge)
 {
+  INFO_LOG(PIXELENGINE, "VIDEO Backend raises INT_CAUSE_PE_TOKEN (btw, token: %04x)", _token);
+  s_Token.store(_token);
```

Comment:
Me neither, but the alternatives are as bad:
- sending also the non-interrupt tokens by events, this is the behavior of master
- only sending the interrupt tokens by events, so they might arrive out-of-order

I see, this is a race condition, but isn't this also a race condition on the real hardware? The token doesn't block the GPU.